client/components/themes-list/docs/example.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ThemesList from 'components/themes-list';
+
+const onScreenshotClickFn = function( theme, index ) {
+	console.log( `Theme ${ theme.id } at ${ index } clicked` );
+};
+
+const ThemesListExample = React.createClass( {
+	displayName: 'ThemesList',
```

Comment:
So the issue here is that the `displayName` is used in determining what components are to be displayed.

1. When you click the header link for any one component it uses the name of the component that _**you**_ supply as part of the `<a href="/devdocs/app-components/`**themes-list**`">` link provided. This is then passed down as the `this.props.component` to the `app-components.jsx` as seen in `client/devdocs/controller.js`. This is the _**selected**_ component.
2. Now in `client\devdocs\design\app-components.jsx --> Collection.shouldWeHide()`, the Collection determines which children components need to be displayed based on whether the `displayName` of any of the children components matches the name of the above selected component.
3. In our scenario, what we end up with is: `themeslist` does not match `themeslistexample` and this results in the component being hidden.

To solve this, we have two options:

1. Keep the `<a href>` header link in line with the path of where the component is defined e.g., `<a href="/devdocs/app-components/themes-list">` and then set the displayName of the Example class to `ThemesList`.

 **Note:** This seems to be the methodology that is being followed (as seen in other component examples).
2. Keep the `displayName` as `ThemesListExample` and update the `<a href>` path to `themes-list-example`.

 **Note:** I find this method cleaner/better, but it will be different from the other examples i've seen previously.

I'll submit my change following Option 2, we can further discuss if there are any other better options.